config/media-library.php

```diff
@@ -48,6 +48,7 @@
      * The fully qualified class name of the model used for temporary uploads.
      *
      * This model is only used in Media Library Pro (https://medialibrary.pro)
+     * @phpstan-ignore-next-line
```

Comment:
I don't like `@phpstan-ignore-next-line` in a config file, could you add the issues that this generates in the phpstan base line.

Also add all other issues that you're now solving with `@phpstan-ignore-next-line` to the baseline 👍